src/quo2/components/list_items/preview_list.cljs

```diff
@@ -0,0 +1,115 @@
+(ns quo2.components.list-items.preview-list
+  (:require [quo.react-native :as rn]
+            [status-im.i18n.i18n :as i18n]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.icon :as quo2.icons]
+            [quo2.components.avatars.user-avatar :as user-avatar]
+            [quo2.components.markdown.text :as quo2.text]))
+
+(def params {32 {:border-radius {:circular 16 :rounded  10}
+                 :hole-radius   {:circular 18 :rounded  12}
+                 :margin-left   -8
+                 :hole-size     36
+                 :hole-x        22
+                 :hole-y        -2}
+             24 {:border-radius {:circular 12 :rounded  8}
+                 :hole-radius   {:circular 13 :rounded  9}
+                 :margin-left   -4
+                 :hole-size     26
+                 :hole-x        19
+                 :hole-y        -1}
+             16 {:border-radius {:circular 8 :rounded  8}
+                 :hole-radius   {:circular 9 :rounded  9}
+                 :margin-left   -4
+                 :hole-size     18
+                 :hole-x        11
+                 :hole-y        -1}})
+
+;; TODO - Add avatar components for other types once implemented
+(defn avatar [item type size]
+  (case type
+    :user [user-avatar/user-avatar
+           (merge {:ring?             false
+                   :status-indicator? false
+                   :size              (case size 32 :small 24 :xs 16 :xxxs)}
+                  item)]))
+
+(defn list-item [idx type size item list-size margin-left
```

Comment:
consider changing variable `idx` to `index`. IMO character saving like this does not add any value and it's better to just avoid potential confusion.

Reply:
hi @J-Son89, Thank you very much for reviewing PR, changed idx to index.